src/arkode/arkode_mristep_nls.c

```diff
@@ -400,7 +405,7 @@ int mriStep_NlsLSetup(booleantype jbad, booleantype* jcur, void* arkode_mem)
   This routine wraps the ARKODE linear solver interface 'solve'
   routine for use by the nonlinear solver object.
   ---------------------------------------------------------------*/
-int mriStep_NlsLSolve(N_Vector b, void* arkode_mem)
+SUNNlsStatus mriStep_NlsLSolve(N_Vector b, void* arkode_mem)
```

Comment:
As with the above routine, the returned values here are not valid `SUNNlsStatus` constants.